src/app/modules/content/learning-objects/question/question.component.scss

```diff
@@ -9,14 +12,12 @@ button {
   margin: 10px;
 }
 
-@import "../../../../../../node_modules/@angular/material/theming";
-
 .color-correct {
-  color: mat-color(mat-palette($mat-indigo))
+  color: #f5222d
```

Comment:
I would put more lighter colors. Roughly, if the background of the dark theme is 80% black (i.e., RGB values 255*0.2), then the red and blue luminosity should be around 40% (between 255 - the white background and 0.2*255). 

The short of it is - make it 10-20% brighter so that it stands out on the black surface more, while still being visible on the white.